### DISCUSSION TITLE
Can I use Civit AI models in Fooocus?

### DISCUSSION THREAD

**Discussion by Paul-Greg** [civitai/civitai#1043] [Q&A]
Hello, people,

Thank you very much in advance.

If yes, how can I make them work?

Best.

**Comment by lonewolf21551** [comment]
Yes. Fooocus uses SDXL checkpoints, so on Civit AI you search for SDXL checkpoints and LoRA . You download checkpoints to the Fooocus\models\checkpoints folder and LoRA to the Fooocus\models\loras folder.
Hmm . . Now I see the date of the post. Well, maybe someone else would need this answer.
